class ConnectionError(Exception):
    pass


class AuthenticationError(Exception):
    pass
